scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -519,12 +536,8 @@ <h5 class="modal-title" id="exampleModalLabel">Edit synopsis</h5>
 
       $('select[multiple]').selectpicker({
         width: '100%'
-        });
       });
 
-      $('select[multiple]').multiselect({
-        buttonWidth: '100%'
-      });
```

Comment:
No, check my comment above. But this code it's not longer used because the multiselect now is using the selectpicker and has its own css specifics, that's why I'm removing it